### PR TITLE
Add blog post for release v1.3

### PR DESCRIPTION
**Pull request by kevin-wangzefeng**
Signed-off-by: Kevin Wang <kevinwzf0126@gmail.com>
Co-authored-by: kadisi <iamkadisi@163.com>
Co-authored-by: Dave Chen <dave.chen@arm.com>
Co-authored-by: fisherxu <xufei40@huawei.com>

* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

Add blog post for release v1.3

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: content/zh/blog/release-v1.3/index.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+在后续的版本中，社区进一步优化云端组件的高并发支持，提高大规模边缘节点场景下的吞吐性能。
+
+
+### 边缘节点自动签发证书
+在1.3版本中，KubeEdge提供了面向边缘节点的证书自动签发能力，简化了用户配置云边安全通道的操作步骤，提升易用性。
+
+默认情况下，KubeEdge会为用户生成自签名证书，用于云端组件和边缘节点的加密通信。针对需要统一管理证书的场景，用户也可以使用指定信任机构签发的证书。
+
+下一步，社区将支持节点的证书自动更新。
```
Comment (kadisi) [inline content/zh/blog/release-v1.3/index.md:90]: 自动更新 -》 过期自动更新？

File: content/zh/blog/release-v1.3/index.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+文档链接：https://github.com/kubeedge/kubeedge/blob/master/docs/setup/kubeedge_configure.md
+
+### 更多的容器运行时支持
+KubeEdge v1.3版对CRI-O和Kata Container进行了进行了全量验证，提供官方集成支持。
```
Comment (kadisi) [inline content/zh/blog/release-v1.3/index.md:95]: 提供官方集成支持-》提供官方集成配置说明

Comment (kevin-wangzefeng) [reply content/zh/blog/release-v1.3/index.md:95]: “提供官方集成支持” 意味着后面有任何使用上的问题，社区都会尝试在KubeEdge或上游项目中fix

File: content/zh/blog/release-v1.3/index.md (hunk 1 above)
Comment (kevin-wangzefeng) [reply content/zh/blog/release-v1.3/index.md:90]: 可以改成到期自动更新，过期就太晚了。
```suggestion
下一步，社区将支持节点的证书到期自动更新。
```

**Pull request merged**